src/pages/graphql/schema/cart/queries/pickup-locations.md

```diff
@@ -10,7 +10,8 @@ Use the `pickupLocations` query to retrieve a list of available pickup locations
 
 The `pickupLocations` query supports the following types of filters.
 
-*  Area filters allow you to specify a location and a radius to find all locations within that radius.
+*  Area filters allow you to specify a location and a radius to find all locations within that radius. The `search_term` field must contain a either the region, city, or postcode, followed by a colon and the country code. This value determines the location to use as the center of the search radius.  Valid search terms include Texas:US, Austin:US, and 78740:US. The two-letter country code must be uppercase.
```

Comment:
```suggestion
*  Area filters allow you to specify a location and a radius to find all locations within that radius. The `search_term` field must contain either the region, city, or postcode, followed by a colon and the country code. This value determines the location to use as the center of the search radius.  Valid search terms include Texas:US, Austin:US, and 78740:US. The two-letter country code must be uppercase.
```

Reply:
I was tempted to change "postcode" to "postal code" until I saw the "postcode" field in the reference.